module/VuFind/src/VuFind/View/Helper/Root/Record.php

```diff
@@ -414,13 +414,12 @@ public function getSearchResult($view)
      *
      * @return string
      */
-    public function getCheckbox($idPrefix = '', $formAttr = false)
+    public function getCheckbox($idPrefix = '', $formAttr = false, $number = null)
     {
-        static $checkboxCount = 0;
```

Comment:
The purpose of the static counter here was originally to ensure that no two labels will end up having the same unique identifier, since previously there were some situations where checkboxes for the same ID would appear multiple places on the same page, and then duplicate IDs would cause HTML validation errors. Perhaps the code has moved on since then and it's no longer a problem, but we should make sure before removing it.

Reply:
The only place I saw this being used was to be set for the template as "count" which then isn't used in the template. The code may have moved on when the number started being added in list-list instead of checkbox.

Reply:
Ahh, I see -- this probably became obsolete when the `<label>` got wrapped around the `<input>`. I think in the past the label and input were separate, and they needed to use an HTML unique ID to link together. Probably best at this stage to remove the count thing entirely since it serves no purpose. If we need some other data element for the HTML, perhaps we should introduce a new parameter. Either way, probably best to do this work in master instead of release-4.1; I don't think it's critical enough to risk an unexpected BC break.